Jenkinsfile

```diff
@@ -31,7 +30,7 @@ pipeline {
             BUILD_TYPE = "Debug"
           }
         }
-        sh "cmake -DUSE_PCH=ON -DBOOST_ROOT=${BOOST_167} -DOPENSSL_ROOT_DIR=${OPENSSL_111} -DSQLITE3_ROOT_DIR=${SQLITE_3253} -DSOPHIATX_STATIC_BUILD=ON -DCMAKE_BUILD_TYPE=${BUILD_TYPE} -DCMAKE_INSTALL_PREFIX=install -DSOPHIATX_EGENESIS_JSON=${GENESIS_FILE} -DBUILD_SOPHIATX_TESTNET=${params.build_as_testnet} -DPRIVATE_NET=${params.build_as_privatenet}"
+        sh "cmake -DUSE_PCH=ON -DBOOST_ROOT=${BOOST_167} -DOPENSSL_ROOT_DIR=${OPENSSL_111} -DSQLITE3_ROOT_DIR=${SQLITE_3253} -DSOPHIATX_STATIC_BUILD=ON -DCMAKE_BUILD_TYPE=${BUILD_TYPE} -DCMAKE_INSTALL_PREFIX=install -DSOPHIATX_EGENESIS_JSON=${GENESIS_FILE} -DBUILD_SOPHIATX_TESTNET=${params.build_as_testnet}"
         sh 'make -j4'
```

Comment:
so there is no PRIVATE_NET flag ? Shouldn't it be separated like: either you build private_net, or test_net or live_net ? Also config parameters might be separated like that ?

Reply:
Not anymore. Just two types of binaries - testnets and production nets, regardless if they are mainnet or private nets. One binary to rule them all... ;)